src/SprykerEco/Client/Payone/PayoneDependencyProvider.php

```diff
@@ -21,6 +21,7 @@ class PayoneDependencyProvider extends AbstractDependencyProvider
      *
      * @return \Spryker\Client\Kernel\Container
      */
+    //phpcs:ignore
```

Comment:
* It's better to not have them.

Reply:
It implements external interface which can't be changed